src/main/java/me/gnat008/perworldinventory/groups/GroupManager.java

```diff
@@ -66,10 +86,10 @@ public void addGroup(String name, List<String> worlds) {
      * @param worlds A list of world names in this group.
      * @param gamemode The default GameMode for this group.
      */
-    public void addGroup(String name, List<String> worlds, GameMode gamemode) {
+    public void addGroup(String name, List<String> worlds, List<String> useLastWorld, List<String> useLastPosInGroup, List<String> useLastPosInWorld, String defaultWorld, GameMode gamemode) {
```

Comment:
Ah, thanks for the naming scheme. I've been staring at this code for a few days now and hit roadblock after roadblock in my coding, and since I need this functionality for my own server, the frustration was building up, making my "how to come up with good names"-part of my brain non-functional 😛